spec/integration.rb

```diff
@@ -94,6 +94,11 @@
   Process.wait(spawn('which hg', :err => :out, :out => '/dev/null'))
   has_mercurial = $?.success?
 
+  if (developer_bin = `xcode-select -p 2>/dev/null`.strip) && $?.success?
+    developer_bin = Pathname(developer_bin) + 'usr/bin'
+    ENV['PATH'] = "#{developer_bin}#{File::PATH_SEPARATOR}#{ENV['PATH']}"
```

Comment:
As this has global side-effects, I would also put it in the global scope at the beginning of the file.